R/slopes.R

```diff
@@ -143,6 +143,7 @@ slope_matrices = function(m_xyz_split, fun = slope_matrix_weighted, ...) {
 #'   datasets. Default: `"bilinear"`.
 #' @param fun The slope function to calculate per route,
 #'   `slope_matrix_weighted` by default.
+#' @importFrom methods is
```

Comment:
Aha, makes sense! Many thanks for a great fix :tada: will aim to look at the other issues next week.